packages/post-rector/src/Collector/UseNodesToAddCollector.php

```diff
@@ -174,7 +174,7 @@ public function getShortUsesByFileInfo(SmartFileInfo $smartFileInfo): array
     /**
      * @return AliasedObjectType[]|FullyQualifiedObjectType[]
      */
-    public function getUseImportTypesByNode(Node $node): array
+    private function getUseImportTypesByNode(Node $node): array
```

Comment:
it is strange that it changed to `private`, as the rule only change to `protected`, it seems the re-use of `getChildrenOfClass()` make it not passed.

Reply:
fixed by skip final class